### PR TITLE
Add static localhost.yml

### PR DESCRIPTION
**Pull request by jhadvig**
@stevekuznetsov PTAL

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
Let's also get the `group_vars/OSEv3/general.yml` and `nested_group_mappings` in this PR -- the `host_vars` you have look right.

**Comment by jhadvig** [comment]
@stevekuznetsov updated.. thanks

File: inventory/group_vars/OSEv3/general.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+openshift_pkg_version: "-0.0.1"
```
Comment (stevekuznetsov) [inline inventory/group_vars/OSEv3/general.yml]: We just merged a PR into Origin that means we don't always build `origin-0.0.1` RPMs -- you'll want to test this out and see if you just want to get rid of this line, or figure out how to get the correct version in here.

Comment (jhadvig) [inline inventory/group_vars/OSEv3/general.yml]: Based on what we pick the version ?

Comment (stevekuznetsov) [inline inventory/group_vars/OSEv3/general.yml]: Previously it was _only_ possible to build Origin RPMs at version `0.0.1`. Now we build more correct versions. I think we should set `openshift_release=v1.5` and make sure we are installing the RPMs that we built and not some other RPMs on accident

**Comment by jhadvig** [comment]
@stevekuznetsov I've updated the PR.. this iventory configuration work, though some additional openshift-ansible tinkering has to be done, but will submit a separate pr for that. 
PTAL

**Comment by stevekuznetsov** [comment]
Do we not want 
```yml
origin_release: 1.5
```

Somewhere in the `OSEv3` configuration?

**Comment by jhadvig** [comment]
Wasnt sure if its necessary, since the version was obtained from built packages

**Comment by stevekuznetsov** [comment]
I think it just pulls latest, which happens to be what we just built. However, the value of this may determine some installer logic -- @sdodson could you clarify if we are losing anything by not specifying `origin_release` ?

**Comment by sdodson** [comment]
> I think it just pulls latest, which happens to be what we just built. However, the value of this may determine some installer logic -- @sdodson could you clarify if we are losing anything by not specifying origin_release ?

origin_release doesn't exist, openshift_release does and it's primarily used to set the version of containerized installations because those aren't bound by yum repo content so we need to know your intended version when we pull the first image. After the origin or ose image is pulled we determine it's full version and then match all subsequent image versions to that image. Unless you care about containerized installs I don't think this matters right now.

**Comment by sdodson** [comment]
PR LGTM

**Pull request merged**